### DISCUSSION TITLE
Two letters in Slovincian

### DISCUSSION THREAD

**Discussion by TheKnightWho** [psb1558/Junicode-font#139] [General]
Hi - the Wiktionary community are a bit puzzled by two letters used in Friedrich Lorentz's dictionary of the Slovincian language and I just wanted to flag them up to you, as we can't find much about them anywhere. You can see the list of letters used for the transcription on [page II of volume 1](https://archive.org/details/slovinzischeswr00loregoog/page/n11/mode/2up).

It's a Latin-based transcription, but with additions from both Greek and Cyrillic. There are Latin versions of most of these (e.g. Latin chi ꭓ U+AB53). However, the two that are proving a problem are:

- ђ (Cyrillic dje), which I think would be properly called "Latin small letter heng with stroke".
- A "w" and "f" overlaid. I can't find anything about this anywhere.

The main problem that we're having is that Slovincian was not a written language for the most part, and Lorentz's transcription is by far the most complete record of the language. As far as we're aware it was his own invention, though, so it's tricky to know how to handle it.

Would it please be possible for you to include these in some fashion? I'm also contacting Michael Everson about this, as it would be good to get his opinion on this as well.

**Comment by psb1558** [comment]
The h-like letter doesn't look much like heng to me, but rather like Junicode's h + cv16. It's got an anchor that lets you attach the short cross stroke U+0335:
<img width="40" alt="image" src="https://user-images.githubusercontent.com/16215767/173693547-6fbb8da4-dbe3-47a3-8a70-f2a333b20ab6.png">
But I could see using the heng if it's more appropriate phonetically. Just in case I'll add an anchor to it so you can  attach the short cross stroke to it.

Looks as if it will also be helpful to add anchors to the IPA shapes of gamma and chi, but for the most part the diacritics are pretty standard and should attach properly. Let me know if there are any problems.

Man, that w with overlaid long s is one ugly glyph! If it's used only in this one book (or in that and Lorentz's Slovincian grammar), it might be good to simply make do in some way, e.g. with w + U+1DE5:
<img width="43" alt="image" src="https://user-images.githubusercontent.com/16215767/173694503-510ea7a8-7fae-4237-8499-9093f4d93ddf.png">

What are you doing about this n with curl?
<img width="43" alt="image" src="https://user-images.githubusercontent.com/16215767/173695372-93abcb6d-f27d-45c6-a90a-e959095fdd1d.png">

**Comment by TheKnightWho** [comment]
Thanks for the response.

I've had a further look into this, and I think it's a situation where Latin equivalents of Cyrillic characters would need to be encoded in Unicode (in a similar fashion to the Greek letters like ꭓ, as you say). If you look at [Vol. I p. 224](https://archive.org/details/slovinzischeswr00loregoog/page/224/mode/2up) of the _Wörterbuch_ you can see that ђ is capitalised as Ђ. Likewise, [Vol. II p. 1244](http://bibliotekacyfrowa.eu/dlibra/doccontent?id=7830) shows ћ capitalised as Ћ - it seems to have been omitted from the list in error.

I *think* it's an "f" overlaid on the "w" rather than a long s, but I did have the same thought as you. I hadn't found [this much higher quality copy](http://bibliotekacyfrowa.eu/dlibra/doccontent?id=7830) when I wrote my initial message which makes it obvious that all "f"s have tiny cross strokes, and the alphabetic ordering suggests that it's supposed to be related to "f" in some fashion - just as ђ "dje" is placed after the other "d" letters. It is very ugly though, you're right.

The n with curl seems to be ꬻ (U+AB3B). Latin Extended-E lists it as a letter used in German dialectology, so it would make sense.

**Reply by psb1558** [reply]
Have you got a link to vol. 1 of the dictionary at http://bibliotekacyfrowa.eu?

So that horrible w with f overlay is like an fw ligature--which is how I'd handle it. Would it work to use f followed by a modifier letter w (apply sups to w). This would be searchable as fw.
<img width="75" alt="image" src="https://user-images.githubusercontent.com/16215767/173870376-f836e5c4-3ab3-4fed-9433-acb0b7c6f190.png">
The capital on p. 224 looks like a TH ligature. You can't easily reproduce this with capital Heng, which looks like this:
<img width="54" alt="image" src="https://user-images.githubusercontent.com/16215767/173871105-fb0b42f1-ba86-4e7f-a935-517361f990c5.png">
What sound is this representing? (BTW, wondering if it would be possible (or worthwhile) to automate the conversion of Lorentz's phonetic system to IPA.)

I don't have U+AB3B, but it would be easy to add.

Can we put together a list of all the things you need to have added for this project?

**Reply by TheKnightWho** [reply]
Sorry - I had meant to link you to vol. 1 in my earlier comment, but mucked it up. Here you go: http://bibliotekacyfrowa.eu/dlibra/doccontent?id=7982.

**Reply by TheKnightWho** [reply]
I have come across [_The Extinct East-Slovincian Kluki-dialect_](https://www.google.co.uk/books/edition/The_Extinct_East_Slovincian_Kluki_dialec/JoCxAAAAIAAJ?hl=en&gbpv=1&bsq=slovincian+kluki&dq=slovincian+kluki&printsec=frontcover), which also uses the combined wf - though Stokhof is obviously just following Lorentz's lead. Having looked a bit further, Lorentz clarifies that it only crops up as an allophone in (what he calls) the Kluckener and Holzkathen-Scholpiner dialects.
![image](https://user-images.githubusercontent.com/6895210/174135159-e6ca4410-44e4-4740-9193-8043e4f72bcc.png)
There's also [_Slovinzische Texte_](http://bibliotekacyfrowa.eu/dlibra/doccontent?id=7829) by Lorentz, which uses it on p. 16.

**Reply by psb1558** [reply]
Stokhof does it a w with a combining f above--which you can do already in Junicode if you want with U+1DEB:
<img width="64" alt="image" src="https://user-images.githubusercontent.com/16215767/174146907-55e1a509-05ef-4781-bf6a-8c5aa8d19569.png">
I can hardly express how bad I think Lorentz's glyph is. Overlaying an f on a w obscures the cross stroke of the f, making that part of the thing ambiguous (not seeing a cross stroke, I first interpreted it as a long s). Stokhof's rendering is much clearer, but ~~IPA for a voiced bilabial fricative is β~~. I meant to say, IPA for an **unvoiced** bilabial fricative is φ.

**Reply by TheKnightWho** [reply]
Thanks - I have suggested that as an alternative. I completely agree with you about Lorentz's glyph, so I'm not surprised that no-one else has used it.

My guess is that it was simply cheaper to print it that way, as I can understand why he's chosen to use a combination of the two.

**Comment by psb1558** [comment]
There's an explanation of these signs in Lorentz's [grammar](https://pbc.gda.pl/dlibra/publication/2056/edition/899/content). The fw is an unvoiced bilabial spirant. He uses  two crossed h letters in the grammar, for unvoiced and voiced affricates, as follows (from p. 119):
<img width="659" alt="image" src="https://user-images.githubusercontent.com/16215767/174070489-2df54e8a-7cf1-48b8-8abc-d05b97919ad5.png">
So I suspect he's trying to come as close to a th ligature as his typefont allows him to come and using fairly arbitrary typographical distinctions to express things like voiced/unvoiced.

**Reply by TheKnightWho** [reply]
Excellent - thank you for finding that. My German is passable but not brilliant (compounded the obsolete technical language!).

The description of ђ and ћ as "vorderpalatal" makes me think he's referring to alveolo-palatal affricates, such as the unvoiced Serbian ћ. I may be barking up the wrong tree, though.

**Reply by psb1558** [reply]
Yes, I think you're right.
